src/pages/Environments.js

```diff
@@ -202,48 +206,51 @@ const Environments = () => {
   const disksByProject = _.groupBy('googleProject', disks)
   const appsByProject = _.groupBy('googleProject', apps)
 
-  const renderBillingProjectApp = app => {
-    const inactive = !_.includes(app.status, ['DELETING', 'ERROR', 'PREDELETING']) &&
-      getCurrentApp(app.appType)(appsByProject[app.googleProject]) !== app
-    return h(Fragment, [
-      app.googleProject,
-      inactive && h(TooltipTrigger, {
-        content: 'This billing project has multiple active cloud environments. Only the most recently created one will be used.'
-      }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
-    ])
-  }
+  // We start the first output string with an empty space because empty space would
+  // not apply to the case where appType is not defined (e.g. Jupyter, RStudio).
+  const forAppText = appType => !!appType ? ` for ${_.capitalize(appType)}` : ''
 
-  const renderBillingProjectRuntime = runtime => {
-    const inactive = !_.includes(runtime.status, ['Deleting', 'Error']) &&
-      getCurrentRuntime(runtimesByProject[runtime.googleProject]) !== runtime
+  const getWorkspaceCell = (namespace, name, appType, shouldWarn) => {
     return h(Fragment, [
-      runtime.googleProject,
-      inactive && h(TooltipTrigger, {
-        content: 'This billing project has multiple active cloud environments. Only the most recently created one will be used.'
+      h(Link, { onClick: () => Nav.goToPath('workspace-notebooks', { namespace, name }) }, [name]),
```

Comment:
I expected this to just go to the Workspace (dashboard), not the Notebooks tab.

Reply:
Also, instead of suppling onClick, can you do the following?

`h(Link, { href: Nav.getLink('workspace-notebooks', { namespace, name }) }, [name]),`

The reason is that my suggested code will render an anchor tag, whereas yours renders a div with aria-role button. This link is taking you someplace, not opening a popup/dialog. Therefore the anchor tag is more meaningful and will behave correctly with assistive technologies.

Reply:
My assumption was, if someone is on `Cloud Environments` page then the chances are, they're using interactive analysis tools, and would want to go to the corresponding (`Notebooks`) tab. It feels `Dashboard` tends to be less relevant for interactive analysis purposes. I can take this up with our UX/PO and change it if they also suggest going to `Dashboard`. What do you think?

Reply:
Sure, run it by UX. My thought is that I don't use the Notebooks tab… I only use the hidden Analysis tab. To me, just getting to the correct Workspace is a big win (though before part of the pain was that the names were obscured).

For the purposes of getting this to "done", feel free to remove the link altogether.

Reply:
I love the link! I'll have them point to `Dashboard` for the time being. 

Thanks for the `href` comment.